subprojects/internal-performance-testing/src/main/groovy/org/gradle/performance/generator/GroovyDslFileContentGenerator.groovy

```diff
@@ -106,16 +101,7 @@ class GroovyDslFileContentGenerator extends FileContentGenerator {
     @Override
     protected String configurationsIfMissingJavaLibrarySupport(boolean hasParent) {
```

Comment:
I think this method should now be called differently. Maybe something like `addJavaLibraryConfigurationsIfNecessary`?